pkg/network/ebpf/c/prebuilt/usm.c

```diff
@@ -572,11 +572,17 @@ static __always_inline int do_sys_open_helper_exit(struct pt_regs* ctx) {
         return 0;
     }
 
-    // Detect whether the file being opened is a shared library
+// Detect whether the file being opened is a shared library
+//    libssl.so
+// libcrypto.so
+// libgnutls.so
```

Comment:
```suggestion
// Detect whether the file being opened is a shared library
//    libssl.so
//    libcrypto.so
//    libgnutls.so
```

Reply:
https://github.com/DataDog/datadog-agent/pull/17723/files#diff-8bf058f88e83ba804fbf856811ff307987754ec40a302d72709e5f34e1091898R588-R592

Reply:
you're missing my intention here, you added indentation on purpose because it looks obvious for you, but as an external reader it is not obvious what you're trying to show here.